src/playdate/sound.nim

```diff
@@ -148,6 +154,15 @@ proc setFinishCallback*(this: FilePlayer, callback: PDFilePlayerCallbackFunction
     else:
         playdate.sound.fileplayer.setFinishCallback(this.resource, privateFilePlayerFinishCallback, cast[pointer](this))
 
+proc fadeVolume*(this: FilePlayer, left, right: float32, len: int32, callback: PDFilePlayerCallbackFunction) =
+    privateAccess(PlaydateSound)
+    privateAccess(PlaydateSoundFileplayer)
+    this.fadeVolumeCallback = callback
+    if callback == nil:
+        playdate.sound.fileplayer.fadeVolume(this.resource, left.cfloat, right.cfloat, len.cint , nil, nil)
+    else:
+        playdate.sound.fileplayer.fadeVolume(this.resource, left.cfloat, right.cfloat, len.cint , privateFilePlayerFadeVolumeCallback, cast[pointer](this))
```

Comment:
The api docs don't mention that last parameter:

https://sdk.play.date/2.6.0/Inside%20Playdate%20with%20C.html#f-sound.fileplayer.fadeVolume

Have you tried this?

Reply:
Yes, in Wheelsprung I use both with and without the last param.

Reported here:
https://devforum.play.date/t/c-docs-fadevolume-is-missing-userdata/20245